css-brochure-form/styles.css

```diff
@@ -0,0 +1,78 @@
+#box {
+  display: flex;
+  justify-content: center;
+  border: white;
+  border-width: 100px;
+  padding-left: 100px;
```

Comment:
This is a pretty high padding. Are you using this to center the content or something to that effect? There are likely other, better ways this can be accomplished 🤓